docs/reference/sql/limitations.asciidoc

```diff
@@ -105,8 +105,8 @@ When multiple values are returned for a field, by default, {es-sql} will throw a
 === Sorting by aggregation
 
 When doing aggregations (`GROUP BY`) {es-sql} relies on {es}'s `composite` aggregation for its support for paginating results.
-However this type of aggregation does come with a limitation: sorting can only be applied on the key used for the aggregation's buckets. 
-{es-sql} overcomes this limitation by doing client-side sorting however as a safety measure, allows only up to *512* rows.
+However this type of aggregation does come with a limitation: sorting can only be applied on the key used for the aggregation's buckets.
+{es-sql} overcomes this limitation by doing client-side sorting however as a safety measure, allows only up to *65535* rows.
```

Comment:
We should backport this to reflect the change that happened quite some time ago.